src/ui/preview/PreviewHeader.vue

```diff
@@ -111,19 +107,21 @@ export default {
     };
   },
   watch: {
-    actionCollection(actionCollection) {
-      if (this.actionCollection) {
-        this.unlistenToActionCollection();
-      }
+    currentView() {
+      // wait for view to render with next tick
+      this.$nextTick(() => {
```

Comment:
we should update this to `await nextTick()` and import it from the `vue` library